compiler/ilgen/IlBuilder.cpp

```diff
@@ -770,6 +770,34 @@ IlBuilder::VectorLoadAt(TR::IlType *dt, TR::IlValue *address)
    return returnValue;
    }
 
+/**
+ * @brief IlBuilder::StructFieldAddress IlGen to get the address of a struct field
+ * @param structName is the name of the struct type the field belongs to
+ * @param fieldName is the name of the field itself
+ * @param obj is an IlValue representing the instance (address) of the struct
+ * @return IlValue representing the address of the field in the struct
+ */
+TR::IlValue *
+IlBuilder::StructFieldAddress(const char* structName, const char* fieldName, TR::IlValue* obj) {
```

Comment:
I would prefer a more descriptive name, such as `StructFieldAddressWithinInstance` to make it clear what the address is relative to.